store/tools/ics23/go.mod

```diff
@@ -4,10 +4,10 @@ go 1.18
 
 require (
 	github.com/confio/ics23/go v0.7.0
-	github.com/cosmos/cosmos-sdk v0.46.0-rc3
+	github.com/cosmos/cosmos-sdk v0.46.0-beta2.0.20220719225934-818da709f20f
```

Comment:
It should be reverted to rc3,  or it uses beta2

Reply:
ok, thank you!